temporalio/envconfig.py

```diff
@@ -0,0 +1,342 @@
+"""Environment and file-based configuration for Temporal clients.
+
+This module provides utilities to load Temporal client configuration from TOML files
+and environment variables.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Any, Mapping, Optional, Union
+
+from typing_extensions import TypeAlias
+
+from temporalio.bridge.temporal_sdk_bridge import envconfig as _bridge_envconfig
+from temporalio.service import ConnectConfig, TLSConfig
```

Comment:
Unless we're intentionally re-exporting, usually when we reference ourselves in code, we do fully-qualified pieces (e.g. `import temporalio.service` + `temporalio.service.ConnectConfig`). Granted that's not always the case, but I think is clearer.

Reply:
done